master-galera/master.cfg

```diff
@@ -231,12 +231,18 @@ for os in os_info:
         else:
             factory = f_deb_build
 
+        env = {}
+        if os == 'ubuntu-2004':
+            print('using gcc-10 for s390')
```

Comment:
this will enable it for all arches: amd64, aarch64, ppc64le and s390x. If it's only needed for s390x, the check should be `if buildern_name == "gal-s390x-ubuntu-2004"`.

Reply:
was just considering gcc-10 for all. Which yes, makes comment incorrect.